index.html

```diff
@@ -172,141 +172,377 @@ <h1 class="main-heading">Work Flow</h1>
       <!-- REVIEWS SECTION -->
 
       <div id="review" class="container review-class">
-        <div class="text-part">
-          <h1 class="main-heading">Reviews or Testimonials</h1>
-          <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent
-            sodales mauris turpis, sed maximus magna egestas vitae. Vestibulum
-            lobortis ligula rhoncus diam consectetur, vitae efficitur nibh
-            fringilla. Sed gravida porttitor molestie. Sed pellentesque lectus
-            sed lacinia accumsan. Sed condimentum arcu risus, non convallis eros
-            tincidunt et. Curabitur elementum mi vitae elit accumsan aliquet.
-            Nam pellentesque molestie pulvinar. Nullam id mauris posuere,
-            sagittis lectus quis, commodo ante. Lorem ipsum dolor sit amet,
-            consectetur adipiscing elit. Morbi faucibus eros metus, sit amet
-            volutpat lorem bibendum ac.
-          </p>
+        <h2 class="main-heading">Reviews</h2>
```

Comment:
Dont remove the `<div class="text-part">` from here. Keep the same format everywhere